python/sdk/Dockerfile

```diff
@@ -12,5 +12,5 @@ COPY . .${WORKDIR}
 
 ARG VERSION
```

Comment:
nit: this arg is not used anymore. we can remove it here and also on build command in github action jobs.